vhdbuilder/packer/install-dependencies.sh

```diff
@@ -289,6 +289,15 @@ else
     retagContainerImage "docker" ${watcherFullImg} ${watcherStaticImg}
 fi
 
+# doing this at vhd allows CSE to be faster with just mv
+cniUnTarNClean() {
```

Comment:
```suggestion
unpackAzureCNI() {
```

or similar (not picky but no stop-n-shop style abbreviation please)

Reply:
in an interview when they ask about my weaknesses. The answer is naming things.